test_program/block.c

```diff
@@ -76,6 +96,8 @@ get_block(int pos)
 
         fstat(fd.fd_block, &st);
         size = st.st_size;
+        REQUIRE(size>0);
+        check_expected(pos);
```

Comment:
check_expected() calls are for mocked functions, not the real ones.  Only the PBC functions (like REQUIRE, ENSURE, etc) are in the real functions.